test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -251,6 +249,51 @@ spvTest iot nio = testCaseSteps "spv client tests" $ \step -> do
         , "target-chain-id" A..= tid
         ]
 
+allocationTest :: IO (Time Integer) -> IO ChainwebNetwork -> TestTree
```

Comment:
One thing is we need an equivalent of `PactTestsSpec` that runs all the repls in `/pact`. We can do that in a separate PR but we really need it.

Reply:
I have one running in `Chainweb/Test/CoinContract.hs`

Reply:
We do this for coin contract already in `Chainweb/Test/CoinContract`